pkg/controller/search/search.go

```diff
@@ -66,6 +67,17 @@ func (s *Search) LatestImage(ctx context.Context, imageURL string, opts *api.Opt
 	return lastestImage.(*api.ImageTag), nil
 }
 
+func (s *Search) ResolveSHAToTag(ctx context.Context, imageURL string, imageSHA string) (string, error) {
+
+	tag, err := s.versionGetter.ResolveSHAToTag(ctx, imageURL, imageSHA)
+	if err != nil {
+		fmt.Println("failed to resolve the sha " + err.Error())
```

Comment:
We never use ‘fmt.Printf’, and instead use a loft which maintains context.

I don’t think a log here is necessary anyway.